decryptDefaultKey/decryptDefaultKey.ino

```diff
@@ -0,0 +1,82 @@
+#include <AESLib.h>
+
+#define BAUD 9600
+
+AESLib aesLib;
+String kim="r9Huf1ov92qFJMtG";
+// AES Encryption Key
+byte aesKey[] = {kim[0], kim[1], kim[2], kim[3], kim[4], kim[5], kim[6], kim[7], kim[8], kim[9], kim[10], kim[11], kim[12], kim[13], kim[14], kim[15]};
+// General initialization vector (use your own)
+byte aesIv[16] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+
+// Generate IV (once)
+void aesInit() {
+  aesLib.gen_iv(aesIv);
+  aesLib.set_paddingmode((paddingMode)0);
+  // encrypt("AAAAAAAAAA", aes_iv); // workaround for incorrect B64 functionality on first run... initing b64 is not enough
```

Comment:
is this comment required/ relevant? if not please remove

Reply:
will remove